contrib/node/examples/src/node/web-component-button/BUILD

```diff
@@ -1,4 +1,5 @@
 node_module(
+  name='web-component-button',
```

Comment:
Just FYI: by not explicitly declaring a name, this was the default target for this directory. That doesn't mean it doesn't have a name: just that its name is equal to the directory name.

As an example, spelling it out explicitly is fine though.